src/flate/reader.cr

```diff
@@ -29,10 +29,10 @@ class Flate::Reader
     @end = false
   end
 
-  # Creates an instance of Flate::Reader, yields it to the given block, and closes
-  # it at its end.
-  def self.new(input : IO, sync_close : Bool = false, dict : Bytes? = nil)
-    reader = new input, sync_close: sync_close, dict: dict
+  # Creates a new reader from the given *io*, yields it to the given block,
+  # and closes it at its end.
+  def self.open(io : IO, sync_close : Bool = false, dict : Bytes? = nil)
```

Comment:
Just mention that signature looks better with:

```crystal
def self.open(io : IO, sync_close : Bool = false, dict : Bytes? = nil) : Nil
```

It's fine for compiler and fine for documentation.

Reply:
just to mention (nested mention ^^), `sync_close : Bool = false` is quite redundant to specify the type.. I think this could be avoided without downside on the documentation.